SYCL/Plugin/sycl-partition-info.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN:  %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// RUN: %RUN_ON_HOST %t.out
```

Comment:
```suggestion
// RUN: %HOST_RUN_PLACEHOLDER %t.out
```
different placeholder is used for llvm-test-suite

Reply:
done